pubspec.yaml

```diff
@@ -16,7 +16,7 @@ environment:
 dependencies:
   logging: ^0.11.0
   meta: ^1.0.0
-  opentracing: ^0.2.3
+  opentracing: ^1.0.1
```

Comment:
I think I messed up the rebase and need to adjust this range to be `">=0.2.3 <2.0.0"`.